modules/core/client/config/core.client.routes.js

```diff
@@ -44,8 +44,7 @@
           }
         },
         data: {
-          ignoreState: true,
-          pageTitle: 'Not Found'
```

Comment:
Agree, i'm not fan of state with '-' ;)
but to manage this, we can replace - by space for title.